repository/SmalltalkCI-Squeak-Core.package/SmalltalkCISqueak.class/class/chooseOptionFrom.title..st

```diff
@@ -0,0 +1,6 @@
+helpers
+chooseOptionFrom: aList title: aString
+
+	^ (Project uiManager respondsTo: #chooseOptionFrom:title:)
+		ifFalse: ["< ToolBuilder-Kernel-tpr.157" Project uiManager chooseFrom: aList title: aString]
+		ifTrue: [Project uiManager chooseOptionFrom: aList title: aString]
```

Comment:
Hm, `Project class >> #uiManager` did not yet exist in Squeak 4.5, so better revert to the old version ...

```suggestion
	^ (UIManager default respondsTo: #chooseOptionFrom:title:)
		ifFalse: ["< ToolBuilder-Kernel-tpr.157" UIManager default chooseFrom: aList title: aString]
		ifTrue: [UIManager default chooseOptionFrom: aList title: aString]
```